query/src/main/resources/application.conf

```diff
@@ -20,12 +20,13 @@ hmda {
 }
 
 db {
-  driver = "slick.driver.H2Driver$"
+  driver = "slick.driver.PostgresDriver$"
 
   db {
-    url = "jdbc:h2:mem:test1;DB_CLOSE_DELAY=-1"
-    driver=org.h2.Driver
-    connectionPool = disabled
-    keepAliveConnection = true
+    # url = "jdbc:postgresql://localhost/hmda?user=postgres&password=postgres"
```

Comment:
I left it because it answers the question that @schbetsy is asking above, about how to set up the `JDBC_URL` environment variable. Kind of obfuscated and it would be better to have docs I suppose